### DISCUSSION TITLE
Regarding Corrupted message

### DISCUSSION THREAD

**Discussion by jaideep11061982** [ultralytics/yolov5#4060] [Q&A]
Can any one help understand why do we see the messages for some images to be as corrupted ?
why yolo finds them corrupt

**Comment by glenn-jocher** [comment]
@jaideep11061982 https://github.com/ultralytics/yolov5/blob/f7d85620601f4c2513bcd2b7911c20fbc49e9097/utils/datasets.py#L847-L857
